Keas.Mvc/ClientApp/components/Workstations/AssignWorkstation.tsx

```diff
@@ -100,29 +100,43 @@ export default class AssignWorkstation extends React.Component<IProps, IState> {
                     onSelect={this._onSelectPerson}
                   />
                 </div>
+                {!this.state.workstation &&
                 <div className="form-group">
-                  <label>Pick a workstation to assign</label>
                   <SearchWorkstations
                     selectedWorkstation={this.state.workstation}
                     onSelect={this._onSelected}
                     onDeselect={this._onDeselected}
                     space={this.props.space}
                   />
-                  </div>
-                {(!this.state.workstation || !this.state.workstation.teamId) && // if we are creating a new workstation, edit properties
+                  </div>}
+                {(this.state.workstation && !this.state.workstation.teamId) && // if we are creating a new workstation, edit properties
```

Comment:
@laholstege i'll allow it but if we were to start over this stuff can all go into their own functions.  Too much conditional logic and comments inside of the render function